src/types.rs

```diff
@@ -287,18 +293,23 @@ impl Into<&str> for &MediaType {
             MediaType::H265 => MEDIA_TYPE_H265,
             MediaType::AAC => MEDIA_TYPE_AAC,
             MediaType::TTXT => MEDIA_TYPE_TTXT,
+            MediaType::VP9 => MEDIA_TYPE_VP9,
         }
     }
 }
 
 #[derive(Debug, PartialEq, Clone, Copy)]
 pub enum AvcProfile {
-    AvcConstrainedBaseline, // 66 with constraint set 1
-    AvcBaseline,            // 66,
-    AvcMain,                // 77,
-    AvcExtended,            // 88,
+    AvcConstrainedBaseline,
+    // 66 with constraint set 1
+    AvcBaseline,
+    // 66,
+    AvcMain,
+    // 77,
+    AvcExtended,
+    // 88,
     AvcHigh,                // 100
-                            // TODO Progressive High Profile, Constrained High Profile, ...
+    // TODO Progressive High Profile, Constrained High Profile, ...
```

Comment:
Is this an auto reformatting thing?

Reply:
Yes, I use IDEA auto reformatting. I will undo it to reduce unnecessary changes.